src/tito/builder/main.py

```diff
@@ -197,6 +197,20 @@ def _create_build_dirs(self):
             mkdir_p(d)
         self._check_build_dirs_access(build_dirs)
 
+    def _copy_extra_sources(self):
+        """
+        Copy extra %{SOURCEX} files to the SOURCE folder.
+        """
+        with open(self.spec_file, 'r') as spec:
+            for line in spec.readlines():
```

Comment:
I am afraid this will not work when there are macros in the file name. I would rather execute `spectool -S self.spec_file` and parse this output.

Reply:
Good call, thank you. Updated.